eclipse/src/saros/ui/wizards/ConfigurationWizard.java

```diff
@@ -30,7 +36,30 @@
 public class ConfigurationWizard extends Wizard {
 
   private final EnterXMPPAccountWizardPage enterXMPPAccountWizardPage =
-      new EnterXMPPAccountWizardPage();
+      new EnterXMPPAccountWizardPage() {
+        @Override
+        public void createControl(final Composite parent) {
```

Comment:
Add a javadoc/comment addressing what this does and why it is necessary?

Reply:
The magic happens on the last line of this method. We are "injecting" something into this composite. Guess I will pull the comment on line 44 up here.

Reply:
Yeah I read the comment but I couldn't really tell what exactly this hack does and why we need it. The commit message hints at it but I still think it wouldn't hurt to spell it out somewhere in the code.